src/components/CreateCoursePage/CreateCoursePage.scss

```diff
@@ -0,0 +1,25 @@
+@import "~bootstrap/scss/_functions.scss";
+@import "~bootstrap/scss/_variables.scss";
+
+.create-course-form {
+  label {
+    font-weight: 600;
+  }
```

Comment:
Maybe `@extend .font-weight-bold`, to standardize weights. (or find a way to add that class to the parent element directly in our React code to avoid the custom sass?)